src/corehost/packaging/dir.props

```diff
@@ -36,7 +36,7 @@
     <HostFullVersion>$(HostVersion)-$(PreReleaseLabel)-$(BuildNumberMajor)</HostFullVersion>
   </PropertyGroup>
 
-  <Target Name="ValidateArgs">
+  <Target Name="ValidateArgs" Condition="'$(CLIBuildVersion)'==''">
```

Comment:
It is really there to have two modes and not force me to pass all these other host properties in when I'm building just the meta-package.